public/css/style.css

```diff
@@ -18,11 +27,74 @@ main {
     padding: 5px 10px;
 }
 
+h1 {
+    color: white;
+}
+
 a {
     text-decoration: none;
 }
 
+nav a {
+    color: white;
+    padding-left: 30px;
+    padding-right: 30px;
+}
+
+nav a:hover {
+    background-color: black;
+    padding: 27.5px;
+    color: white;
+}
+
 .profile-small-section {
     height: 170px;
     overflow: auto;
 }
+
+.grid-container {
+    display: grid;
+    grid-template-areas: 
+        'header-left header-left header-left header-right header-right header-right'
+        'left-one left-one center-one center-one right-one right-one';
+    grid-gap: 10px;
+    padding: 10px;
+}
+
+.grid-item-1 {
+    grid-area: header-left;
+    font-size: 30px;
+    text-align: center;
```

Comment:
https://getbootstrap.com/docs/4.0/utilities/text/
Please refer to the class names about text-align predefined in bootstrap

Reply:
I'm not using bootstrap for making the grid. That's just custom css.